external-crates/move/crates/move-ir-to-bytecode-syntax/src/syntax.rs

```diff
@@ -741,6 +756,11 @@ fn parse_term_(tokens: &mut Lexer) -> Result<Exp_, ParseError<Loc, anyhow::Error
             consume_token(tokens, Tok::RParen)?;
             Ok(Exp_::ExprList(exps))
         }
+        Tok::At => {
+            tokens.advance()?;
+            let address = parse_account_address(tokens)?;
+            Ok(Exp_::address(address).value)
```

Comment:
What were address literals before? 0x? 
We should probably change that some day lol

Reply:
Yea, they were (/are) still all `0x...` and there's some weird thing we do to try and determine if it should be an address or not...

Totally agreed that we should change it someday, but I didn't feel in such a desire of pain today to make that change in this PR.